### PR TITLE
Zib-PulseRate

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

File: resources/zib/zib-PulseRate.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    <uri value="https://zibs.nl/wiki/PulseRate-v3.3(2020EN)" />
+    <name value="zib PulseRate-v3.3(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/pattern-VitalSigns" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
```
Comment (LuudSlagter) [inline resources/zib/zib-PulseRate.xml:32]: The comment on the root that is derived from the pattern-VitalSigns profile should be changed, as the text is not correct in this case (PulseRate not being an 'official' vital sign based on core FHIR). The comment should at least indicate that we've chosen to base PulseRate on the pattern-VitalSigns profile due to it being closely related to HeartRate (and thus also for consistency), even though it is not included in the core VitalSigns profile.

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: resources/zib/zib-PulseRate.xml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-PulseRate.xml:86]: Deleted slice and added pattern

**Comment by LuudSlagter** [comment]
@niekvangalen Could you review my last commit regarding the root comment?

**Review by niekvangalen — APPROVED** [review]
Nice work, approved!

**Pull request merged**